app.py

```diff
@@ -299,28 +299,63 @@ def user_logged():
 
 
 # users view
-@app.route('/users', methods=['GET'])
+@app.route('/users', methods=['POST'])
```

Comment:
Why POST request?

Reply:
@L04DB4L4NC3R Mistakenly preassume that get request wan't have jwt token,
Needed to make it accessable for only admin.
Will change back to get request.
Get request takes headers right?